### PR TITLE
Update Javlibrary Python (sceneByName)

### PR DESCRIPTION
**Pull request by Belleyy**
### You need the latest dev version or wait for the 0.10 release.

Only search on Javlibrary not r18.

### CODE REVIEW

**Comment by bnkai** [comment]
Assuming `JAV.py` in the yaml file is actually `JavLibrary_python.py` i get 
```
DEBU[2021-09-20 00:06:19] Scraper script <python3 JavLibrary_python.py searchName> started 
ERRO[2021-09-20 00:06:19] [Scrape / JavLibrary Python] Traceback (most recent call last): 
ERRO[2021-09-20 00:06:19] [Scrape / JavLibrary Python]   File "/home/username/.stash/scrapers/JavLibrary_python.py", line 401, in <module> 
ERRO[2021-09-20 00:06:19] [Scrape / JavLibrary Python]     scene_url = FRAGMENT["url"] 
ERRO[2021-09-20 00:06:19] [Scrape / JavLibrary Python] KeyError: 'url' 
```
From what i see
```python
FRAGMENT = json.loads(sys.stdin.read())

scene_url = FRAGMENT["url"]
```
the `scene_url` assignment shouldnt happen in the sceneByName case

File: scrapers/JavLibrary_python.py [hunk 1]
```diff
@@ -407,6 +418,9 @@ def th_imagetoBase64(imageurl, typevar):
 jav_main_html = None
 r18_main_html = None
 
+if "validName" in sys.argv and SCENE_URL is None:
```
Comment (bnkai) [inline scrapers/JavLibrary_python.py:421]: i don't see a `SCENE_URL` anywhere, maybe `scene_url` ?

**Comment by bnkai** [comment]
There is still an issue when trying to do a search by sceneByName when the scene already has a URL (IMO when doing a search by name for a scene the scene url if present should be ignored)
The `if SCENE_URL: `(line 426) is triggered  ( could be avoided by a `SEARCH_TITLE is not None` )
That is not enough though as you get 
```
ERRO[2021-09-23 00:50:01] [Scrape / JavLibrary Python] [DEBUG] Directly the movie page (https://www.javlibrary.com/en/?v=xxxxxxxxx) 
ERRO[2021-09-23 00:50:01] could not unmarshal json: json: cannot unmarshal array into Go struct field ScrapedScene.title of type string 
```
I think the `if jav_search_html:` (line 478) might need some more tweaking ?

**Comment by Belleyy** [comment]
ohh yea didn't think about that. 
I think i got the solution but look like every site is protected by cloudflare ATM, so can't really test it.

**Comment by bnkai** [comment]
Still getting the error when doing a query search
```
DEBU[2021-09-26 12:08:12] Scraper script <python3 JavLibrary_python.py searchName> started 
ERRO[2021-09-26 12:08:13] [Scrape / JavLibrary Python] [DEBUG] Using search with Title: SRS-022 
ERRO[2021-09-26 12:08:13] [Scrape / JavLibrary Python] [DEBUG] === Checking Status of Javlib site === 
ERRO[2021-09-26 12:08:13] [Scrape / JavLibrary Python] [javlibrary] Using this site for scraping (200) 
ERRO[2021-09-26 12:08:13] [Scrape / JavLibrary Python] [DEBUG] ====================================== 
ERRO[2021-09-26 12:08:13] [Scrape / JavLibrary Python] [DEBUG] Directly the movie page (https://www.javlibrary.com/en/?v=javlijtptm) 
ERRO[2021-09-26 12:08:13] could not unmarshal json: json: cannot unmarshal array into Go struct field ScrapedScene.title of type string
```

**Pull request merged**